examples/tutorial/tutorial_notebooks/introduction/exercise_3.ipynb

```diff
@@ -291,6 +291,7 @@
     "# Hint: Look at the kurtosis walkthrough above\n",
     "\n",
     "modin_mad_custom = ...\n",
+    "print(modin_mad_custom)\n",
```

Comment:
Printing is needed to materialize DataFrames to correctly measure execution time and pass assert statement bellow.